utils/tracing/go/tracing.go

```diff
@@ -0,0 +1,104 @@
+// Package tracing provides a simple utility for including opentelemetry and zipkin tracing
+// instrumentation in vHive and Knative workloads
+package tracing
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/exporters/trace/zipkin"
+	"go.opentelemetry.io/otel/propagation"
+	"go.opentelemetry.io/otel/sdk/resource"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	semconv "go.opentelemetry.io/otel/semconv"
+	"go.opentelemetry.io/otel/trace"
+)
+
+func initTracer(tp *sdktrace.TracerProvider) func() {
+	otel.SetTracerProvider(tp)
+	otel.SetTextMapPropagator(propagation.NewCompositeTextMapPropagator(propagation.TraceContext{}, propagation.Baggage{}))
+	return func() {
+		_ = tp.Shutdown(context.Background())
+	}
+}
+
+func newZipkinExporter(url string, logger *log.Logger) *zipkin.Exporter {
```

Comment:
upon an error, this function should log a warning (not fatal) and return the error to the caller to handle it